googletest/include/gtest/internal/gtest-internal.h

```diff
@@ -1282,21 +1286,6 @@ class FlatTuple
   }
 };
 
-class AdditionalMessage {
- public:
-  AdditionalMessage(const char* message) : value(message) {}
-  AdditionalMessage& operator=(const std::string& message) {
-    value = message;
-    return *this;
-  }
-  operator bool() const { return ::testing::internal::AlwaysTrue(); }
```

Comment:
Just a note that the previous code here used AlwaysTrue(), I don't know what significance that has so I left the ConstCharPtr implementation as it was with a simple "return true;"

Reply:
I don't know either, I just kept the AlwaysTrue() from the previous if statement